html/pages/index.html

```diff
@@ -26,88 +26,108 @@
   </nav>
 
   <section class="hero">
-    <h1 class="logo">We create beautiful products.</h1>
+    <div class="container">
+      <div class="row">
+        <h1 class="logo">We create beautiful products.</h1>
 
-    <p>
-      Truffaut DIY keffiyeh, twee messenger bag venmo
-      locavore organic master cleanse marfa gochujang selvage.
-    </p>
+        <p>
+          Truffaut DIY keffiyeh, twee messenger bag venmo
+          locavore organic master cleanse marfa gochujang selvage.
+        </p>
 
-    <a href="#" class="btn" role="button">Learn More</a>
+        <a href="#" class="btn" role="button">Learn More</a>
 
-    <a href="#" class="btn" role="button">Play video</a>
+        <a href="#" class="btn" role="button">Play video</a>
 
-    <a href="#services" class="btn" role="button"></a>
+        <a href="#services" class="btn" role="button"></a>
```

Comment:
It is very special button, so we don't need bootstrap defaults for it. Please, remove the `btn` class, and add both class and id like `scroll-down-button`